core/src/OpenACC/Kokkos_OpenACC_ParallelReduce_Team.hpp

```diff
@@ -258,7 +268,10 @@ KOKKOS_INLINE_FUNCTION void parallel_reduce(
 #define KOKKOS_IMPL_ACC_REDUCE_TEAM_PRAGMA \
   num_workers(team_size) vector_length(vector_length)
 #define KOKKOS_IMPL_ACC_REDUCE_TEAM_ITRS league_size
-#define KOKKOS_IMPL_ACC_REDUCE_TEAM_LEAGUE_ID_INIT i
+#define KOKKOS_IMPL_ACC_REDUCE_TEAM_MEMBER_INIT                          \
```

Comment:
I think this macro is only use in one place (or I missed it?)

Reply:
The macro `KOKKOS_IMPL_ACC_REDUCE_TEAM_MEMBER_INIT` is defined in multiple places (in line 155, 161, and 271), but used only in one place (in line 405).